beetmoverscript/src/beetmoverscript/gcloud.py

```diff
@@ -203,6 +203,7 @@ async def push_to_releases_gcs(context):
 
     # Weed out RELEASE_EXCLUDE matches, but allow partners specified in the payload
     push_partners = context.task["payload"].get("partners", [])
+    exclude = context.task["payload"].get("exclude", []) + list(RELEASE_EXCLUDE)
```

Comment:
It isn't a problem now, but I will note that this will subject VPN pushes to the existing `RELEASE_EXCLUDE` patterns, which could theoretically be a problem in the future. Hopefully we'll have removed this hardcode by the time any such thing comes up.

Reply:
From my understanding of the beetmover scripts, that was always the case and I felt that this approach was the best way to minimize the potential for unintended changes in behaviour. Once this PR lands we can probably work towards removing `RELEASE_EXCLUDE` by migrating the patterns into their associated tasks.

The other way we could have done this would be to use `RELEASE_EXCLUDE` as the default value for existing tasks that rely on `RELEASE_EXCLUDE` to omit files, for example:
```
exclude = context.task["payload"].get("exclude", list(RELEASE_EXCLUDE))
```

Reply:
You're right - this is already the case. Sorry for the confusion. I think what you have here is fine, but the above suggestion would be fine as well, if you want to go that route.